lib/menu.rb

```diff
@@ -0,0 +1,29 @@
+class Menu
+
+  attr_reader :food_holder
+
+  def initialize
+    @food_holder = []
+  end
+
```

Comment:
The convention would be to call an array of dishes @dishes instead of @food_holder